### ISSUE TITLE
Duplicate preferred names

### ISSUE THREAD

**Issue by poshest** [whosonfirst-data/whosonfirst-data#1898]
Are there supposed to be multiple preferred names for a given names id for a given language (and region?)? Surely only one should be preferred and the rest moved to type 'variant'?

```
select id, region, count(*), min(name)
from names 
where privateuse = 'preferred'
    and language = 'eng'
group by 1, 2
having count(*) > 1;
```

```
85675745		2	Gilgit Baltistan
85682615		2	Araba / Álava
404474405		2	Gerwen en Nederwetten
1108936149		2	Cottage District
1108936771		2	Gum Grove
1108936785		2	Hunters Village
1108955941		2	QIM
1126110289		2	Guadeloupe
1126110311		2	Mont-Bleu
1126110317		2	Jean-Proulx
1126131137		2	Stationsbuurt
1126131139		2	Tuinen en Frisia
1126131151		2	Bloemketerp en Zevenhuizen
1126131217		2	't War
1141963615		2	Langen en Zwiep
1141963675		2	Berm en Hoorzik
1141963683		2	Verspreide huizen Velddriel
1141963941		2	Bijsteren en Hoef
1141964187		2	De Pol en omgeving
1141964227		2	De Keyzer en omgeving
1141964233		2	De Eng en omgeving
1141964587		2	Balijnenland en Bronsbergen
1141965371		2	's-Heerenberg
1158783689		3	Henxel
1158783775		2	Linde
1158787021		2	Waddenlaan
1158787033		3	Hoefslag
1158787037		2	Camping de Houtrib
1158787039		2	Jagersbos
1158787045		4	Damrif
1158787047		2	Bongerd
1158787049		3	Hofstede
1158787055		3	Beginweg
1158787057		2	Overijsselsehout
1158787061		3	Badweg
1158787063		2	Lelycentre
1158787085		2	Golfpark
1158787111		3	Fortuijn
1158787119		2	Meerdijkhaven
1158787121		3	Galjoen
1158787145		2	Houtribsluizen
1158787151		2	Ijsselmeer
1158787153		2	Houtribbos
1158787157		2	Houtribbos
1158787195		3	Gelderse Hout
1158787199		2	Voorstraat
1158787201		2	Getijdenbuurt
1158787203		3	Archipel
1158787241		3	Bosvalkweg
1158787263		2	Kofschip
1158787275		2	De Doelen
1158787277		2	De Meierij
1158787293		2	Het Ravelijn
1158787297		2	Ijsselmeerdijk
1158787299		4	Bijl-
1158787301		4	Edelhertwg
1158787303		3	Dronterwg
1158787311		2	Larserbos
1158787315		3	Eendenweg
1158787317		3	Meeuwenweg
1158787321		3	Larserweg
1158791137		2	Onderwijs-
1158796911		2	Elzen en Neerbroek
1158797003		2	Luissel en Nergena
1158797027		2	Bosch en De Scheeken
1158797461		2	Eliasterrein
1158797533		2	Bennekel-West
1158797569		2	Noord 4
1158797577		2	Baai 4
1158797605		2	Achter de Molen
1158797631		2	't Winhof
1158797823		2	Raam en Molenstraat
1158798353		2	Opwetten
1158798895		2	Beekkant en Beekvliet
1158798967		2	Laar en Zandhoef
1158800105		3	Bosch
1158802109		3	't Zwarte
1158802293		2	Lasonder
1158803663		2	De Beitel
1158805625		2	Egede
1158807611		3	Billinghuizen en Waterop
1158818883		2	5 en 6
1158819495		2	Indische buurt en omgeving
1158819499		2	Heemsteedse Dreef
1158820261		2	Opmeer-Zuid
1158820457		2	Hoornder Nieuwland
1158820873		2	Drieban (gedeeltelijk)
1158820877		2	Drieban (gedeeltelijk)
1158821017		3	Munniksveen en omgeving
1158821135		2	Schepenwijk
1158823209		2	Hofland
1158825995		2	Burgen- en Steinenbuurt
1158826713		2	Bedrijventerrein Langeweg
1158826787		2	Bebouwde Kom
1158828775		2	Gijbeland
1158829179		2	Bovenberg en Lekdijk
1360666039		3	Holy Mountain
```

Two examples of duplicates taken from the list above.

```
'Gilgit Baltistan'
85675745	region	PK	eng						preferred	Gilgit Baltistan	1582351428
85675745	region	PK	eng						preferred	Gilgit Baltistan Province	1582351428

'Holy Mountain'
1360666039	region	GR	eng						preferred	Mt Athos	1566641406
1360666039	region	GR	eng						preferred	Mt. Athos	1566641406
1360666039	region	GR	eng						preferred	Holy Mountain	1566641406
```

**Comment by stepps00** [comment]
Thanks for the issue @poshest 

> Are there supposed to be multiple preferred names for a given names id for a given language (and region?)? Surely only one should be preferred and the rest moved to type 'variant'?

WOF does allow for more than one preferred name value, but we've settled on trying to maintain a single preferred name per language.

I'm noticing that many of the examples you've listed are admin records we maintain for places in the Netherlands. These records were updated through https://github.com/whosonfirst-data/whosonfirst-data/issues/837. In these cases, it looks like we parsed name values on a `,` value when updating names, resulting in some names being split into multiple preferred values. [Example](https://raw.githubusercontent.com/whosonfirst-data/whosonfirst-data-admin-nl/master/data/115/878/730/1/1158787301.geojson):

```
"wof:name":"Edelhertwg, Runderwg, Wisentwg, Elandwg",
```

```
    "name:nld_x_preferred":[
        "Edelhertwg",
        "Runderwg",
        "Wisentwg",
        "Elandwg"
    ],
```

These issues should be corrected... when we file a PR to fix, we'll link to this issue.
